libcrux-ml-kem/pqcp/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "libcrux-ml-kem"
+version = "0.0.2-pre.2"
```

Comment:
Would it be easier to copy the existing one and `sed` these things?

Reply:
I've included a Python script that builds a `Cargo.toml` from the workspace and crate `Cargo.toml`s. This way, these things should be in sync always.